text/2394-async_await.md

```diff
@@ -115,36 +100,24 @@ async { /* body */ }
 (async || { /* body */ })()
 ```
 
-except that control-flow constructs like `return`, `break` and `continue` are
-not allowed within `body` (unless they appear within a fresh control-flow
-context like a closure or a loop). How the `?`-operator and early returns
-should work inside async blocks has not yet been established (see unresolved
-questions).
+除了像 `return`、`break` 和 `continue` 這樣的控制流結構不允許在 `body` 中使用（除非它們出現在一個新的控制流上下文中，比如 closure 或 loop）。 `?`-operator 和 early return 如何在非同步區塊中工作尚未確定（請參閱未解決的問題）。
 
-As with `async` closures, `async` blocks can be annotated with `move` to capture
-ownership of the variables they close over.
+與 `async` closure 一樣，`async` 區塊可以添加 `move`，以捕獲區塊內所包覆的變數的所有權。
 
-## The `await!` compiler built-in
+## 編譯器內嵌的 `await!`
 
-A builtin called `await!` is added to the compiler. `await!` can be used to
-"pause" the computation of the future, yielding control back to the caller.
-`await!` takes any expression which implements `IntoFuture`, and evaluates to a
-value of the item type that that future has.
+一個名為 `await!` 的內嵌函式被添加到編譯器。`await!` 可用於「暫停」future 的計算，將控制權交還給呼叫者。`await!` 接受任何實作 `IntoFuture` 的表達式，並計算為此 future 所具有的項目型別的值。
```

Comment:
「並計算為此 future 定義的回傳型別的值」is better ?

Reply:
> Not sure about the correct translation of **item**. It refers to language item I beliveve. @CYBAI do you have any idea?

Sorry for late reply 🙏 yeah, I think it refers to the `Output` generic as @FizzyElt mentioned in https://github.com/rust-tw/rfcs-tw/pull/28#discussion_r994272927.

> 「並計算為此 future 定義的回傳型別的值」is better ?

thus, I wonder this would be a good candidate; however, as `Output` is just a name of the generic, maybe we can say 
 `泛型型別` instead of `回傳型別`? and we can refer to the `Output` below

```
並計算為此 future 所傳入的泛型型別（e.g. 下面範例的 `Output`）之值
```

WDYT?

Reply:
Either looks fine! I slightly prefer to @CYBAI's.